tests/test-gadget.vala

```diff
@@ -16,6 +16,7 @@ namespace Frida.GadgetTest {
 					data_dir = Path.build_filename (source_dir, "tests");
 					break;
 				}
+				case Frida.Test.OS.TVOS:
 				case Frida.Test.OS.IOS: {
```

Comment:
Should swap the order here for consistency -- since it's iOS followed by tvOS elsewhere. (Even if this makes the diff larger.)